console/console-init/ui/src/components/MessagingInstances/MessagingInstance/Enmasse/EnmasseAddressSpaces.js

```diff
@@ -23,7 +23,7 @@ const translateAddressSpaces = addressSpaces => {
     return translation;
 }
 
-const getPlanNames = plans => plans.spec.plans.map(plan => plan.name).sort();
+const getPlanNames = plans => plans.spec.plans.map(plan => plan.name);
```

Comment:
This could get the displayName now.

Reply:
When creating the address space, you still need to input the name and not the displayName in the .spec.plan field, so one would have to remember both.